frontend/src/composables/use-audio-snackbar.ts

```diff
@@ -0,0 +1,40 @@
+import { computed, ref } from "vue"
+
+import { useUiStore } from "~/stores/ui"
+
+/**
+ * Composable that handles showing and hiding of the snackbar
+ * with instructions for audio interactions.
+ *
+ * Shows the snackbar when the user focuses the first audio track.
+ * Hides the snackbar when the user interacts with it (seeks or clicks on play button).
+ */
+export const useAudioSnackbar = () => {
+  const isMouseDown = ref(false)
+  const handleMouseDown = () => {
+    isMouseDown.value = true
+  }
+
+  const showSnackbar = () => {
+    if (isMouseDown.value) {
+      // The audio player was clicked to open the single result view, not
+      // focused via keyboard.
+      isMouseDown.value = false
+    } else {
+      useUiStore().showInstructionsSnackbar()
+    }
+  }
+  const hideSnackbar = () => {
+    useUiStore().hideInstructionsSnackbar()
+  }
+  const snackbar = {
+    isVisible: computed(() => useUiStore().areInstructionsVisible),
+    show: showSnackbar,
+    hide: hideSnackbar,
+    handleMouseDown,
+  }
+
+  return {
+    snackbar,
+  }
```

Comment:
Why not remove one object and just return the snackbar instead, so the usage becomes

`const snackbar = useAudioSnackbar()` or `cost { isVisible } = useSnackbar()`? 

I think that pattern better matches our existing composables.